### ISSUE TITLE
Rule Request: [todo] Extend todo rule to allow FIXME and TODO be separately chosen

### ISSUE THREAD

**Issue by grantneufeld** [realm/SwiftLint#5232]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### New rule request

Currently, the `todo` rule issues warnings for both `TODO` and `FIXME` comments. I would like to be able to configure the rule to only apply to one, or both, of those types of comment. (Specifically, in my case, I only want warnings for `FIXME` comments, and no warnings for `TODO`.)

1. Why should this rule be added?

Currently, in order to have SwiftLint produce warnings for one of the types of comments, we have to have warnings issued for both—even in the case where we do not want warnings on the the second type.

2. Provide several examples of what _would_ and _wouldn't_ trigger violations.

Identical to current rule, except warnings would not be issued for the excluded type of comment.

3. Should the rule be configurable, if so what parameters should be configurable?

Yes. This request is to add a configuration key such as `only`, which would take `FIXME` or `TODO` as its value.

```yaml
todo:
  only:
    - FIXME
```

4. Should the rule be opt-in or enabled by default? Why?

The rule is currently enabled by default.
